developer_guides/topology2/topology2.rst

```diff
@@ -1332,6 +1332,51 @@ You can use the ``-P`` switch to convert a 2.0 configuration file to the 1.0 con
 
    alsatplg <-D args=values> -P input.conf -o output.conf
 
+Split topologies
+****************
+
+Linux kernel can load multiple topologies, a topology for a single function.
+This feature is useful when a single device is disabled in specific productuds. To achieve this, you need to split
+the topology into multiple tplg files. The split topology files should be named as follows:
+
+.. code-block:: bash
+
+        sof-<platform>-<function>-id<BE id number>.tplg
+
+Where <platform> should be the platform name like mtl, lnl, etc.
+
+Where <function> should be one of
+
+.. code-block:: bash
+
+        sdca-jack
+        sdca-<n>amp where n is the amp link numbers
+        sdca-mic
+        dmic-<n>ch where n is the channel numbers. Currently, only 2ch and 4ch are supported.
+        sdca-hdmi
+
+
+For example
+
+.. code-block:: bash
+
+        sof-arl-sdca-2amp-id2.tplg
+        sof-arl-sdca-mic-id4.tplg
+        sof-arl-dmic-2ch-id5.tplg
+        sof-arl-sdca-hdmi-id7.tgpl
+
+The split topologies are the subset of the monolithic topology. Usually, you just need to add a description with proper
+macro settings to disable the features that you don't need and set the first BE ID that in the topology in the cmake file
+to generate the split topologies.
```

Comment:
Can these topologies have dependencies in order of loading them, that one topology refers to some component from other topology?
LIke direct routing audio from mic to speaker via some processing.

Reply:
We don't support this so far. Currently, all split topologies are independent.